### PR TITLE
fix(server): check for external urls in array

### PR DESCRIPTION
**Pull request by EslamHiko**
- [x] This is a **bugfix**
- [ ] This is a **feature**
- [ ] This is a **code refactor**
- [ ] This is a **test update**
- [ ] This is a **docs update**
- [ ] This is a **metadata update**

### For Bugs and Features; did you add new tests?
Yes

### Motivation / Use-Case
fixes : https://github.com/webpack/webpack-dev-server/issues/1879

### Breaking Changes
no

### Additional Info
Tests pass then errors show up 😕 maybe because I use windows, let's see will CI be green or not.

### CODE REVIEW

File: test/ExternalPathsWatch.test.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
+'use strict';
```
Comment (evilebottnawi) [inline test/ExternalPathsWatch.test.js]: Move this tests in https://github.com/webpack/webpack-dev-server/blob/master/test/ContentBase.test.js

File: test/ExternalPathsWatch.test.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        expect(e.message).toBe('Watching remote files is not supported.');
+        compiler.run(() => {
+          done();
+        });
+      }
+    });
+  });
+
+  describe('testing single & multiple internal paths', () => {
+    it('Should not throw exception (single line)', (done) => {
```
Comment (evilebottnawi) [inline test/ExternalPathsWatch.test.js]: `single line` -> `string`

File: test/ContentBase.test.js [hunk 3]
```diff
@@ -256,50 +256,59 @@ describe('ContentBase', () => {
...
         expect(e.message).toBe('Watching remote files is not supported.');
-        testServer.close(() => {
-          done();
-        });
+        done();
       }
     });
   });
 
   describe('testing single & multiple internal paths', () => {
```
Comment (EslamHiko) [inline test/ContentBase.test.js]: tests in this part are not necessary, all other tests already test with valid contentBase.

**Comment by EslamHiko** [comment]
@evilebottnawi since we have now `is-absolute-url` what about using it instead of Regex?

**Comment by evilebottnawi** [comment]
@EslamHiko yes, will be great :+1:

**Comment by evilebottnawi** [comment]
Sorry for delay

**Comment by hiroppy** [comment]
Thanks!

**Comment by EslamHiko** [comment]
@evilebottnawi no problems 😃. @hiroppy My pleasure 😄.

**Pull request merged**